frontend/src/component/ui/SearchBar.scss

```diff
@@ -0,0 +1,70 @@
+.search-box {
+  display: flex;
+  position: relative;
+  margin: 0 30px;
+  height: 100%;
+
+  .search-input {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+
+    input {
+      flex: 1;
+    }
+
+    i.search {
+      flex: 0;
+      position: relative;
+      top: 4px;
+      right: 30px;
+      color: black;
+    }
+  }
+
+  .suggestions {
+    display: none;
+    position: absolute;
+    list-style: none;
+    padding: 0;
+    width: calc(100% - 20px);
+    z-index: 1000;
+    margin-top: 44px;
+    border-radius: 4px;
+    line-height: 2;
+    box-shadow: 0 4px 6px 0 rgba(32, 33, 36, 0.28);
```

Comment:
Nit: This should be moved to https://github.com/short-d/short/blob/master/frontend/src/component/styles/shadow.scss